<launch>
    <node pkg="kato_program" type="enet_lidar_fusion2_with_odometry_filtered" name="enet_lidar_fusion2_with_odometry_filtered" output="screen"/>
    <rosparam command="load" file="$(find kato_program)/config/params.yaml" />
</launch>
